ext/bin/require-jdk

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -uexo pipefail
+
+script_home="$(cd "$(dirname "$0")" && pwd)"
+
+default=openjdk10
+
+cmdname="$(basename "$0")"
+
+usage() { echo "Usage: $cmdname VERSION INSTALLDIR_IF_NEEDED"; }
```

Comment:
Not sure, but now that I think about it, wonder if it might be worth a trivial elaboration here, to give an indication of what's expected, i.e. something like this for the usage:

    Usage: $cmdname (openjdk10|default|...) INSTALLDIR_IF_NEEDED